### ISSUE TITLE
No fire activation and build issues w/ 680 and 

### ISSUE THREAD

**Issue by Zitt** [WZG-Mods/wzg-icarus-balance-overhaul#15]
Haven't been able to play given the latest icarus 
REV. 2.1.20.120144-SHIPPING-HYPATIA HYPATIA
and the latest 680 release

Nothing with fire seems to activate (camp fires, stoves, fireplaces) so not able to sleep.
Also; "activated" things seem to be broken. IE scarecrow with spoiled meat does not keep the animals away and show "inactive".

**Comment by Zitt** [comment]
Confirmed still an issue with W700 and latest patch as of 3/29/24

**Comment by Local9** [comment]
Personally not experienced this with this version, have you tested with a new character/world?

**Comment by Zitt** [comment]
No. We have a lot invested in these characters and the world we've built.  We will not be starting over.

**Comment by Local9** [comment]
You're missing the point of what I'm saying. You don't have to delete the character or the world, you can create another prospect, put them on the same server world, and test the issue.
